### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by denysbelei9**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,36 @@
-# write your code here
+
+
+class Animal:
+    alive = []
+
+    def __init__(
+
+
```
Comment (Bohdan-Salamakha) [inline app/main.py:8]: remove these blank lines

File: app/main.py [hunk 2]
```diff
@@ -1 +1,36 @@
...
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name},"
+            f" Health: {self.health},"
+            f" Hidden: {self.hidden}}}")
```
Comment (Bohdan-Salamakha) [inline app/main.py:22]:
```suggestion
        return (
            f"{{Name: {self.name},"
            f" Health: {self.health},"
            f" Hidden: {self.hidden}}}"
            )
```

File: app/main.py [hunk 3]
```diff
@@ -1 +1,36 @@
...
+        self.hidden = not self.hidden
+
+
+class Carnivore(Animal):
+    @staticmethod
+    def bite(herbivore: Herbivore) -> None:
+        if isinstance(herbivore, Herbivore) and not herbivore.hidden:
+            herbivore.health -= 50
+        if herbivore.health <= 0:
+            Animal.alive.remove(herbivore)
```
Comment (Bohdan-Salamakha) [inline app/main.py:36]:
```suggestion
    def bite(herbivore: Herbivore) -> None:
        if isinstance(herbivore, Herbivore) and not herbivore.hidden:
            herbivore.health -= 50
            if herbivore.health <= 0:
                Animal.alive.remove(herbivore)
```

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
v1

File: .flake8 [hunk 4]
```diff
@@ -4,4 +4,4 @@ ignore = E203, E266, W503, ANN002, ANN003, ANN101, ANN102, ANN401, N807, N818
 max-line-length = 79
 max-complexity = 18
 select = B,C,E,F,W,T4,B9,ANN,Q0,N8,VNE
-exclude = venv, tests
+exclude = venv, tests
```
Comment (ArsenPidhoretskyi) [inline .flake8:7]: Return this changes

**Pull request closed without merging**